src/libaktualizr/http/httpclient.cc

```diff
@@ -53,6 +53,9 @@ HttpClient::HttpClient(const std::vector<std::string>* extra_headers) {
 
   curlEasySetoptWrapper(curl, CURLOPT_VERBOSE, get_curlopt_verbose());
 
+  curlEasySetoptWrapper(curl, CURLOPT_FOLLOWLOCATION, 1L);
+  curlEasySetoptWrapper(curl, CURLOPT_MAXREDIRS, 10L);
```

Comment:
This curl param should be set in HttpClient::downloadAsync(). But won't work anyway because aktualizr/package_manager tries to download it three times.